spec/hardware/system.fmf

```diff
@@ -42,4 +45,4 @@ example:
 
 link:
   - implemented-by: /tmt/steps/provision/mrack.py
-    note: "``system.numa-nodes`` and ``system.model-name`` only"
+    note: "``system.vendor`` and ``system.model`` not implemented yet"
```

Comment:
Perhaps I'm misunderstanding, but shouldn't it now be implemented along with `system.numa-nodes`?
```suggestion
    note: "``system.numa-nodes``, ``system.vendor`` and ``system.model`` only"
```